raiden_contracts/tests/fixtures/base/utils.py

```diff
@@ -176,8 +176,9 @@ def get(txn_hash: str, message: Optional[str] = None, additional_gas: int = 0) -
 
 @pytest.fixture()
 def get_block(web3: Web3) -> Callable:
-    def get(txn_hash: HexBytes) -> int:
+    def get(txn_hash: HexBytes) -> BlockData:
```

Comment:
Nitpick (probably out-of-scope for PR): I dunno how well your IDE support for Pytest is. Do you get the signature of this function in the test cases? It kinda confuses me as the name is not very descriptive. My first intention is that this is again just a mimic of `web3.eth.get_block`. But its actual purpose is to get the block for a given transaction hash. Just that nothing is expressing this except the function parameter for the getter of this fixture where IDE support might be week. At least the code where this gets used reads not very nice.
Anyway, I guess you can just close this conversation. But in case your IDE supports reliable renaming for Pytest fixtures, consider to rename it if you agree with me. No further discussion necessary.